src/transactions/TransactionFrame.cpp

```diff
@@ -870,6 +881,12 @@ TransactionFrame::consumeRefundableSorobanResources(
         consumedContractEventsSizeBytes, sorobanConfig, cfg);
     if (feeRefund < consumedFee.refundable_fee)
     {
+        pushSimpleDiagnosticError(
+            cfg, SCE_BUDGET, SCEC_EXCEEDED_LIMIT,
+            "refundable resource fee was not sufficient to cover the events "
+            "fee after paying for ledger storage rent: {} > {}",
```

Comment:
I think we've moved away from "rent" terminology in the docs, instead of "ledger storage rent" what about "storage TTL fee"?

Reply:
Have we though? I thought we're still naming the payment 'rent payment' (and as result it extends TTL). The fees doc still talks about 'paying rent', which makes sense to me.